src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -373,6 +373,16 @@ public boolean isTestFeedback() {
         return this.type == FeedbackType.AUTOMATIC && !isStaticCodeAnalysisFeedback() && !isSubmissionPolicyFeedback();
     }
 
+    /**
+     * Checks whether the feedback was given manually by a tutor
```

Comment:
I think we could also mention that this includes feedback automatically created by Athena and approved by tutors.